portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,9 +65,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long timestamp = (long) entity.getProperty("timestamp");
       String message = (String) entity.getProperty("message");
       FormSubmissions comment = new FormSubmissions(id, fname, lname, email, number, timestamp, message);
+
       comments.add(comment);
-      //commentCount -= 1;
```

Comment:
You can remove this. 

Instead in L57, check if length of  List<FormSubmissions> comments  == commentsCount

Reply:
I deleted my comments since we are making the same point here.